workers/unity/ProjectSettings/ProjectSettings.asset

```diff
@@ -606,7 +606,7 @@ PlayerSettings:
   webGLLinkerTarget: 0
   webGLThreadsSupport: 0
   scriptingDefineSymbols:
-    1: UNITY_POST_PROCESSING_STACK_V2;UNITY_DISABLE_AUTOMATIC_SYSTEM_BOOTSTRAP;IMPROBABLE_WORKERTYPE_UNITYGAMELOGIC;IMPROBABLE_WORKERTYPE_UNITYCLIENT;IMPROBABLE_WORKERTYPE_SIMULATEDPLAYERCOORDINATOR
+    1: UNITY_POST_PROCESSING_STACK_V2;UNITY_DISABLE_AUTOMATIC_SYSTEM_BOOTSTRAP;DISABLE_REACTIVE_COMPONENTS
```

Comment:
why are they only disabled for PC

Reply:
Good question, will disable for all platforms. Might have to do this for playground as well